plugins/dbgate-plugin-mssql/src/backend/sql/functionParameters.js

```diff
@@ -0,0 +1,26 @@
+module.exports = `
+SELECT 
+    o.object_id as parentObjectId,
+    p.object_id AS parameterObjectId,
+    CASE 
+        WHEN p.name IS NULL OR LTRIM(RTRIM(p.name)) = '' THEN 
+            '@Output'
+        ELSE 
+            p.name
+    END AS pureName,
+    TYPE_NAME(p.user_type_id) AS dataType,
+    p.max_length AS charMaxLength,
+    p.precision AS precision,
+    p.scale AS scale,
+    p.is_output AS isOutputParameter,
+    p.parameter_id AS parameterIndex
+FROM 
+    sys.objects o
+JOIN 
+    sys.parameters p ON o.object_id = p.object_id
+WHERE 
+    o.type IN ('FN', 'IF', 'TF')
```

Comment:
add condition (+ join sys.schemas s)
o.object_id =OBJECT_ID_CONDITION and s.name =SCHEMA_NAME_CONDITION

this is neccessary for partial analyse, eg. when single object is requested